actions/build/expansion.py

```diff
@@ -47,16 +44,16 @@ def __init__(self, ai):
         return False
 
     async def handle(self, iteration):
-        if self.worker_to_first_base:
-            self.ai.actions.append(await self.send_worker_to_next_expansion())
-            self.worker_to_first_base = False
+        # if self.worker_to_first_base:
+        #     self.ai.actions.append(await self.send_worker_to_next_expansion())
+        #     self.worker_to_first_base = False
 
-        if self.expand_now:
-            await self.ai.expand_now()
-            return True
+        # if self.expand_now:
```

Comment:
if you add here the contraint `and not len(self.ai.townhalls) < 2` is should fix the bug as well and reduce the amount of changes :)

Reply:
this also fails, i try to find another solution

Reply:
? dont get it but ok